terraform/github/locals.tf

```diff
@@ -21,7 +21,7 @@ locals {
       description = "Core Repository for Data Platform Data Products"
   }]
 
-  ap_migration_apps = slice(jsondecode(file("./ap_migration_apps.json")), 11, 15)
+  ap_migration_apps = slice(jsondecode(file("./ap_migration_apps.json")), 11, 12)
```

Comment:
Can you remove this change, or merge main into your branch? We're no longer taking a slice of the ap_migration_apps.